templates/primary-secondary/demo/integration.html

```diff
@@ -0,0 +1,355 @@
+<!DOCTYPE html>
```

Comment:
This is a new demo page that contains the primary-secondary template along with a couple complicated components.  It is useful to have this page both for testing and demo purposes.  The dialog was removed from the standard demo page.